.devcontainer/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM fedora
+RUN dnf install -y \
+    fish \
+    git \
+    jq \
+    make \
+    qemu-img \
+    osbuild \
+    osbuild-ostree \
+    pylint \
+    python3-autopep8 \
+    python3-devel \
+    python3-docutils \
+    python3-iniparse \
+    python3-jsonschema \
+    python3-pip \
+    python3-pylint \
+    python3-pytest \
+    rpm-build \
+    selinux-policy-devel
```

Comment:
For example, this should be enough:

```suggestion
    selinux-policy-devel
    
RUN useradd developer
USER developer
```

Reply:
I think that also needs a `"remoteUser": "vscode"` in the `devcontainer.json`